assemblies/plugins/databases/sapdb-assemblies/pom.xml

```diff
@@ -18,13 +18,27 @@
     <name>Hop Assemblies Plugins Databases SAP DB</name>
     <description></description>
 
+	<properties>
+    	<sapdbc.version>7.4.4</sapdbc.version>
+	</properties>
 
     <dependencies>
         <dependency>
             <groupId>org.hop</groupId>
             <artifactId>hop-plugins-databases-sapdb</artifactId>
             <version>${project.version}</version>
         </dependency>
+    	<dependency>
+      		<groupId>com.sap</groupId>
+      		<artifactId>sapdbc</artifactId>
+      		<version>${sapdbc.version}</version>
+      		<exclusions>
+        		<exclusion>
+          			<groupId>*</groupId>
+          			<artifactId>*</artifactId>
+        		</exclusion>
+      		</exclusions>
+    	</dependency>        
```

Comment:
I can't find what license this driver is created under, do you happen to know which it is?
If we can't find it we will have to remove this include top avoid future issues

Reply:
License not found, except that: "In October 2007 SAP assumed full sales and commercial support for MaxDB. MaxDB 7.6 is now closed source, available free-of-charge (without support, and with usage restrictions) for use with non-SAP applications."